surveys/micro/2025-06-compiler-performance.md

```diff
@@ -0,0 +1,224 @@
+# Survey questions
+
+The goal of this survey is to figure out how much do Rust users struggle with compilation times and which compilation workflows are the most relevant for them. We would like to use this data so that we can focus the efforts of Rust compiler contributors on the areas of compilation performance that matter the most to our users.
+
+Your feedback is much appreciated. This survey is fully anonymous. The Rust survey team will go through the answers and release a summary on the Rust blog after the survey is complete.
+
+## User classification
+
+### Do you use Rust?
+
+Type: select one
+
+- Yes, I use Rust [`NEXT`](#do-you-code-in-rust-at-a-company)
+- No, I don't currently use Rust, but I have in the past [`NEXT`](#were-long-compilation-times-the-primary-reason-why-you-stopped-using-rust)
+- No, I have never used Rust [`END`](#other)
```

Comment:
Is this option necessary? I'm honestly not interested in hearing feedback from people who have never used Rust.

Reply:
Having some sanity questions to filter on can be useful.
https://en.wiktionary.org/wiki/Lizardman%27s_Constant

Reply:
Right, me neither, but we need to filter these people out. This is just a control question to make sure that people who actually use Rust are answering the survey. If you answer "no", the survey will immediately end for you.